booklist.js

```diff
@@ -0,0 +1,28 @@
+export default class BookList {
+  constructor() {
+    this.list = [];
+  }
+
+  addBook(book) {
+    this.list.push(book);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use array methods such as `concat` over `push`, that doesn't mutate the array.